rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -17,179 +17,156 @@
-import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
-import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+import org.ethereum.config.blockchain.upgrades.*;
 import org.ethereum.core.*;
 import org.ethereum.core.CallTransaction.Function;
 import org.ethereum.crypto.ECKey;
 import org.ethereum.vm.LogInfo;
 import org.ethereum.vm.PrecompiledContracts;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
-import org.junit.jupiter.params.provider.ValueSource;
-
-import java.math.BigInteger;
-import java.time.Instant;
-import java.util.Arrays;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import static co.rsk.peg.bitcoin.BitcoinTestUtils.coinListOf;
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import org.junit.jupiter.params.provider.*;
 
 /**
  * Test class for BridgeEventLoggerImpl.
  *
  * @author martin.medina
  */
-
 class BridgeEventLoggerImplTest {
-
-    private static final BridgeRegTestConstants bridgeRegtestConstants = new BridgeRegTestConstants();
-    private static final FederationConstants federationRegtestConstants = bridgeRegtestConstants.getFederationConstants();
-    private static final NetworkParameters regtestParameters = bridgeRegtestConstants.getBtcParams();
-    private final ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+    private static final BridgeConstants BRIDGE_CONSTANTS = BridgeMainNetConstants.getInstance();
+    private static final FederationConstants FEDERATION_CONSTANTS = BRIDGE_CONSTANTS.getFederationConstants();
+    private static final NetworkParameters NETWORK_PARAMETERS = BRIDGE_CONSTANTS.getBtcParams();
+    private static final BtcTransaction BTC_TRANSACTION = new BtcTransaction(NETWORK_PARAMETERS);
+    private static final RskAddress RSK_ADDRESS = new RskAddress("0x0000000000000000000000000000000000000000");
+    private static final Keccak256 RSK_TX_HASH = RskTestUtils.createHash(1);
 
     private List<LogInfo> eventLogs;
     private BridgeEventLogger eventLogger;
-    private BtcTransaction btcTxMock;
-    private BtcTransaction btcTx;
     private SignatureCache signatureCache;
 
     @BeforeEach
     void setup() {
+        ActivationConfig.ForBlock activations = ActivationConfigsForTest.lovell700().forBlock(0L);
```

Comment:
should we use `all` instead of `lovell`?

Reply:
True, updated